packages/databyss-api/src/routes/api/pages.js

```diff
@@ -209,11 +249,16 @@ router.get(
       let entries = await getEntriesFromId(entriesList)
       entries = dictionaryFromList(entries)
 
+      const topicsList = blockList.filter(b => b.type === 'TOPIC')
```

Comment:
If we refactor to the more generic `populateRefEntities`, we can rewrite this more concisely:
```
const [sources, entries, topics] = ['SOURCE', 'ENTRY', 'TOPIC'].map(t => {
  const list = blockList.filter(b => b.type === t)
  const populated = populateRefEntities(list, t)
  return dictionaryFromList(populated)
})
```